game-core/src/main/java/games/strategy/triplea/delegate/battle/steps/BattleSteps.java

```diff
@@ -168,13 +181,7 @@ public boolean isAmphibious() {
     if (defensiveSubsSubmerge.getOrder() == SUB_DEFENSIVE_RETREAT_BEFORE_BATTLE) {
       steps.addAll(defensiveSubsSubmerge.getNames());
     }
-    // See if there any unescorted transports
-    if (isBattleSiteWater
-        && Properties.getTransportCasualtiesRestricted(gameData)
-        && (attackingUnits.stream().anyMatch(Matches.unitIsTransport())
-            || defendingUnits.stream().anyMatch(Matches.unitIsTransport()))) {
-      steps.add(REMOVE_UNESCORTED_TRANSPORTS);
-    }
+    steps.addAll(removeUndefendedUnits.getNames());
```

Comment:
Is it inconsistent to have 'removeUndefendedUnits' and 'removeUnprotectedUnits'?

Reply:
I changed the name of this class multiple times.  I thought I had updated all of the places with the latest name but it looks like I missed one :)